chat-commands.js

```diff
@@ -128,6 +128,31 @@ exports.commands = {
 			'How many digits of pi do YOU know? Test it out <a href="http://guangcongluo.com/mempi/">here</a>!');
 	},
 
+	code: function (target, room, user) {
+		if (!target) return this.parse('/help code');
+		if (!this.canTalk()) return;
+		const seperator = '\n';
+		if (target.includes(seperator)) {
+			if (!this.runBroadcast('!code')) return;
+			const params = target.split(seperator).map(param => param.split(' ').join('&nbsp;'));
```

Comment:
`param.split(' ').join('&nbsp;')`

umm, this is an incredibly inefficient way to write

`param.replace(/ /g, '&nbsp;')`